packages/treats/package.json

```diff
@@ -86,6 +86,7 @@
         "isbinaryfile": "3.0.3",
         "jest": "23.6.0",
         "jest-transform-graphql": "2.1.0",
+        "jquery": "3.3.1",
```

Comment:
This can be achieved without jQuery, let's not introduce jQuery dependency for just selecting the service worker tag 😄